.github/actions/download-build-artifacts/action.yml

```diff
@@ -0,0 +1,36 @@
+name: "Download build artifacts"
+description: "Download build artifacts"
+runs:
+  using: "composite"
+  steps:
+    ## NXCACHE
+    - name: update NX cache folder
+      uses: actions/download-artifact@v3
+      continue-on-error: true
```

Comment:
why this? it will fail anyway later trying to untar it

Reply:
yep, that's a legacy stuff. removing it now :D